solutions/usaco-740.mdx

```diff
@@ -18,15 +22,25 @@ const int MAX_N = 20;
 vector<vector<char>> image(MAX_N, vector<char>(MAX_N));
 vector<vector<bool>> visited(MAX_N, vector<bool>(MAX_N));
 
-// PCL described by top-left (i1,j1) and bottom-right (i2,j2) corner
+/** PCL delimited by the top left corner (i1, j1) & bottom right corner (i2, j2) */
 struct PCL {
-	int i1, j1, i2, j2;
+	int i1, j1;
+	int i2, j2;
+
+	bool is_inside(PCL o) {
+		return (
+			i1 >= o.i1 &&
+			i2 <= o.i2 &&
+			j1 >= o.j1 &&
+			j2 <= o.j2
+		);
+	}
```

Comment:
better name than o?

Reply:
i mean it's `o` for `other`, should i rename it to that?

Reply:
ya